api/src/prepareDb.ts

```diff
@@ -12,7 +12,7 @@ export async function preDb() {
     "CREATE TABLE IF NOT EXISTS desk (desk_id TEXT PRIMARY KEY, desk_state TEXT)"
   );
   await db.exec(
-    "CREATE TABLE IF NOT EXISTS booking (booking_id TEXT PRIMARY KEY, start_time NUM, end_time NUM, booked_desk TEXT, FOREIGN KEY(booked_desk) REFERENCES desk(desk_id))"
+    "CREATE TABLE IF NOT EXISTS booking (booking_id TEXT PRIMARY KEY, start_time INTEGER, end_time INTEGER, booked_desk TEXT, FOREIGN KEY(booked_desk) REFERENCES desk(desk_id))"
```

Comment:
I expected some feedback here to explain why Integer is the correct your.
The math is something like this: According to the docs the max size is in 8 bytes, signed. We will have only positive numbers, so we can use 8 bytes of 8 bits each so in total 64 bits, minus one for the sign is 63 bits. 2^63 = 9 000 000 000 000 000 000.

That means that the highest date we can store in Unix ms format is hundreds of years in the future, so this should be fine.

However, SQLite also offers special functions to store dates, you can look into it:
 https://www.sqlite.org/lang_datefunc.html